sunshine/confighttp.cpp

```diff
@@ -360,7 +361,8 @@ void saveConfig(resp_https_t response, req_https_t request) {
 
       configStream << kv.first << " = " << value << std::endl;
     }
-    write_file(config::sunshine.config_file.c_str(), configStream.str());
+    std::string sunshine_config_path = SUNSHINE_CONFIG_DIR "/sunshine.conf";
```

Comment:
`config::stream.config_file` is configurable by the user.
Therefore, it shouldn't be changed at runtime